src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/IdentifierFilter.php

```diff
@@ -3,6 +3,7 @@
 namespace Pim\Bundle\CatalogBundle\Elasticsearch\Filter;
 
 use Akeneo\Component\StorageUtils\Exception\InvalidPropertyTypeException;
+use Pim\Bundle\CatalogBundle\Elasticsearch\Filter\Field\AbstractFieldFilter;
```

Comment:
Because it is a field and an attribute. For consistency we decide to put it at the folder's root :)